ee/ucp/admin/install/install-on-azure.md

```diff
@@ -4,6 +4,88 @@ description: Learn how to install Docker Universal Control Plane in a Microsoft
+and Persistent Storage feature set. Because of this there are prerequisites to 
+installing Docker UCP in Azure.
+
+## Azure Prerequisites 
+
+Please find a list of infrastructure pre-requisites that need to be met in order 
+to successfully deploy Docker UCP on Azure. Note Docker provides an automated 
+deployment mechanism which will adhere to these prerequisites
+[here](http://success.docker.com/article/certified-infrastructures-azure).
+
+- All Docker EE Nodes (Managers and Workers) need to be deployed into the same 
+Azure Resource Group. The Azure Networking (Vnets, Subnets, Security Groups) 
+components could be deployed in a second Azure Resource Group.
+- All Docker EE Nodes (Managers and Workers) need to be attached to the same 
+Azure Subnet.
+- All Docker EE Nodes (Managers and Workers) need to be tagged in Azure with the
+tag `Orchestrator=Kubernetes:1.8.11`.
+- All Docker EE Worker Nodes need to be deployed within the Same Availability 
+Set.
+- The Azure Computer Name needs to match the Node Operating System's Hostname. 
+- An Azure Service Principal with `Contributor` access to the Azure Resource 
+Group hosting the Docker EE Nodes. Note, if using a separate networking Resource 
+Group the same Service Principal will need `Network Contributor` access to this 
+Resource Group.
+
+### Retrieve Azure secrets
+
+To continue with the Docker EE deployment you will need to retrieve the 
+following bits of information. 
+
+- `subscriptionId` - The Azure Subscription ID in which the Docker EE 
+objects are being deployed. 
+- `tenantId` - The Azure Active Directory Tenant ID in which the Docker EE 
+objects are being deployed. 
+- `aadClientId` - The Azure Service Principal ID
+- `aadClientSecret` - The Azure Service Principal Secret Key
+
+### Azure Configuration File
+
+For Docker UCP to integrate in to Microsoft Azure, an Azure configuration file 
+will need to be placed within each Docker EE node in your cluster. This file 
+will need to be placed at `/etc/kubernetes/azure.json`.  
+
+See the template below. Note entries that do not contain `****` should not be 
+changed.
+
+```
+{
+    "cloud":"AzurePublicCloud", 
+    "tenantId": "***",
+    "subscriptionId": "***",
+    "aadClientId": "***",
+    "aadClientSecret": "***",
+    "resourceGroup": "***",
+    "location": "****",
+    "subnetName": "/****",
+    "securityGroupName": "****",
+    "vnetName": "****",
+    "primaryAvailabilitySetName": "****",
+    "cloudProviderBackoff": false,
+    "cloudProviderBackoffRetries": 0,
+    "cloudProviderBackoffExponent": 0,
+    "cloudProviderBackoffDuration": 0,
+    "cloudProviderBackoffJitter": 0,
+    "cloudProviderRatelimit": false,
+    "cloudProviderRateLimitQPS": 0,
+    "cloudProviderRateLimitBucket": 0,
+    "useManagedIdentityExtension": false,
+    "useInstanceMetadata": true
+}
+```
+
+There are some optional values for alternative Azure networking deployments.
```

Comment:
When do these apply? vnetResourceGroup if the vnet is in a separate resource group. How about routeTableName? Can we call out when you would specify these.